crates/precompile/src/secp256k1.rs

```diff
@@ -45,20 +45,23 @@ mod secp256k1 {
 #[allow(clippy::module_inception)]
 mod secp256k1 {
     use primitives::{alloy_primitives::B512, keccak256, B256};
-    use secp256k1::{
+    use rust_secp256k1::{
```

Comment:
Hey @stevencartavia, the idea is not to replace it but to add a new option with a feature flag for `rust_secp256k1`.

I would see to try `cfg_if` (there are a few examples of its usage in the repo)